src/routes/index.tsx

```diff
@@ -1,30 +1,13 @@
 import React from 'react'
-import { Route, Link } from 'react-router-dom'
+import { Route } from 'react-router-dom'
+
+// Pages
+import { ExampleHomePage } from '../pages/Home'
+import { ExampleAboutPage } from '../pages/About'
 
 export const EXAMPLE_ABOUT_PAGE_ROUTE_NAME = '/about'
 export const EXAMPLE_HOME_PAGE_ROUTE_NAME = '/'
 
-export const ExampleAboutPage = () => (
```

Comment:
These functions aren't real pages. Since it's just elements to be tested, my main goal when placing them in the same file to be able to easily read them, without needing to jump to different modules. 

Another argument on not splitting these in different modules is that it's a package for testing utilities and not an application with a lot of pages that can scale & thus needs to be structure differently.